MinimedKit/PumpEvents/PlaceholderPumpEvent.swift

```diff
@@ -9,26 +9,33 @@
 import Foundation
 
 
-public struct UnknownPumpEvent: TimestampedPumpEvent {
+public struct PlaceholderPumpEvent: TimestampedPumpEvent {
     public let length: Int
     public let rawData: NSData
     public let timestamp: NSDateComponents
+    public let name: String
```

Comment:
Hah. I find this amusing, because it was my previous style (in minimed_rf) to calculate everything as needed from the data, but then I saw you doing calc on init, and decided I liked that better as well.  Guessing you have a reason for why you consider this different, but still... :)